### ISSUE TITLE
[Bug]: Using Metamask on Mobile With WalletConnect V2 and transactions are suddenly failing to appear

### ISSUE THREAD

**Issue by jordaniza** [MetaMask/metamask-mobile#8226]
### Describe the bug

Using a sveltekit dapp connected to MM mobile via wallet connect v2, when submitting the transaction for signing, the app hands off to metamask, which opens, but no transaction pops up. If I return back to the app, the app is still awaiting a signature. No error occurs.

This was working last week, so not sure what has changed?

### Steps to reproduce

Go to app.ambos.finance on mobile. 

Attempt to initiate a Tx when connected with MM via WC v2.

The app will switch to Metamask when signing a Tx, but nothing will appear to sign.

### Version

7.21.3

### Build type

None

### Device

Google Pixel 6 Pro

### Operating system

Android

### Additional context

Only started happening a few days ago

**Comment by PressGarden** [comment]
Hello,

Thank you for reporting this issue. This has been assigned to the appropriate team to look into.

Best regards, 
Axel

**Comment by christopherferreira9** [comment]
Hi @jordaniza ! Can you please provide the wallet version where you started to encounter this issue? I've just checked main and it is working there so it might be related to that specific version of the wallet.

**Comment by jordaniza** [comment]
Yep:

```json
        "@walletconnect/ethereum-provider": "~2.9.1",
        "@walletconnect/modal": "~2.6.0",
```

**Comment by christopherferreira9** [comment]
I'm sorry, I meant the MetaMask wallet version. To get this you can go to Settings -> About MetaMask

**Comment by A-Feder** [comment]
Hey @jordaniza, can you share a screenshot of your MetaMask version from **Settings -> About MetaMask** to confirm the version? The latest and upcoming MetaMask mobile is 7.14

**Comment by jordaniza** [comment]
Yeah my mistake it's 7.12.3

**Comment by christopherferreira9** [comment]
Hey @jordaniza ! 
I just confirmed that the bug is present on that specific version. I went ahead and double checked the latest release candidate and it appears to be solved there. It'll be fixed by version 7.14.0 which ir right around the corner.

**Comment by jordaniza** [comment]
Awesome thank you guys for looking into this. Appreciate the speedy fix.
